src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -47,19 +51,50 @@ trait DataAccess extends AutoCloseable {
   /** Returns all outputs for this workflowId */
   def getWorkflowOutputs(workflowId: WorkflowId)(implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]]
 
-  def getAllOutputs(workflowId: WorkflowId)(implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]]
+  protected def getAllOutputs(workflowId: WorkflowId)(implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]]
+
+  def getOutputs(workflowId: WorkflowId)(implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]] = {
+    getAllOutputs(workflowId)(ec) map { _ filterNot IsStandardStream }
+  }
 
   def getAllInputs(workflowId: WorkflowId)(implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]]
 
-  /** Get all outputs for the scope of this call. */
+  /** Get all output symbols for the scope of this call including standard stream symbols. */
+  protected def getAllOutputs(workflowId: WorkflowId, key: ExecutionDatabaseKey)
+                             (implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]]
+
+  /** Get all "real" outputs for the scope of this call (i.e. exclude standard stream symbols). */
   def getOutputs(workflowId: WorkflowId, key: ExecutionDatabaseKey)
-                (implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]]
+                (implicit ec: ExecutionContext): Future[Traversable[SymbolStoreEntry]] = {
+    getAllOutputs(workflowId, key)(ec) map { symbols => symbols filterNot IsStandardStream }
+  }
+
+  /** Get the standard stream symbols for the scope of this call. */
+  def getStandardStreamSymbols(workflowId: WorkflowId, key: ExecutionDatabaseKey)
+                              (implicit ec: ExecutionContext): Future[CallLogs] = {
+
+    // Folds stdout and stderr symbols into an accumulator CallLogs object.
+    def foldLogs(acc: CallLogs, symbol: SymbolStoreEntry): CallLogs = {
+      symbol match {
+        case s if s.key.name.endsWith("$stdout") => acc.copy(stdout = s.wdlValue.get.asInstanceOf[WdlFile])
+        case s if s.key.name.endsWith("$stderr") => acc.copy(stderr = s.wdlValue.get.asInstanceOf[WdlFile])
```

Comment:
Should these be `endsWith(".$stdout")`? Elsewhere the magic seems to be '.$' and not just '$'.